tests/foreman/cli/test_contentview.py

```diff
@@ -773,6 +816,55 @@ def test_positive_create_composite(self):
 
     @tier2
     @run_only_on('sat')
+    def test_positive_create_composite_by_name(self):
+        """Create a composite content view and add non-composite content
+        view by its name
+
+        :id: c91271d8-efb8-487e-ab11-2e9e87660d3c
+
+        :expectedresults: Composite content view is created and has another
+            view associated to it
+
+        :BZ: 1416857
+
+        :CaseLevel: Integration
+        """
+        # Create REPO
+        new_repo = make_repository({u'product-id': self.product['id']})
+        # Sync REPO
+        Repository.synchronize({'id': new_repo['id']})
+        # Create CV
+        new_cv = make_content_view({u'organization-id': self.org['id']})
+        # Associate repo to CV
+        ContentView.add_repository({
+            u'id': new_cv['id'],
+            u'repository-id': new_repo['id'],
+        })
+        # Publish a new version of CV
+        ContentView.publish({u'id': new_cv['id']})
+        new_cv = ContentView.info({u'id': new_cv['id']})
+        cvv = new_cv['versions'][0]
+        # Create CV
+        cv = make_content_view({
+            'composite': True,
+            'organization-id': self.org['id'],
+        })
+        # Associate version to composite CV
+        ContentView.add_version({
+            u'content-view-version-id': cvv['id'],
```

Comment:
Hm, test description has two meanings :) 'its name' could be applied to non-composite an composite CV. Anyway, I'd rather use names for both CVs in this test and ids for CVs in `by_id` test for better coverage.